lib/matplotlib/colors.py

```diff
@@ -1047,22 +1044,10 @@ def inverse(self, value):
             return vmin * pow((vmax / vmin), value)
 
     def autoscale(self, A):
-        """
-        Set *vmin*, *vmax* to min, max of *A*.
-        """
-        A = np.ma.masked_less_equal(A, 0, copy=False)
-        self.vmin = np.ma.min(A)
-        self.vmax = np.ma.max(A)
+        super().autoscale(np.ma.masked_less_equal(A, 0, copy=False))
```

Comment:
I thought about it, then decided against because here all methods are in the same file *and* the overrides are quite short (one or two lines), so I thought things were clear enough as they are.  If you really want it let me know (or push it yourself, either way).

Reply:
I would still do it. It's a statement of intent. Without that, someone will be tempted in the future to add a docstring.

Also, let's just make it a general general rule. Every public method should have either a docstring or a `# docstring inherited` comment. If we then encounter a method without that, it's immediately clear that nobody has thought about the documentation so far.

Reply:
done (x4)